File: PYTHON/Ejercicio2.py
'''
Escribe un programa que calcule el tiempo de descarga de un archivo en función del tamaño
del archivo y la velocidad de descarga en megabits por segundo. El programa debe pedir
al usuario que ingrese el tamaño del archivo en megabytes y la velocidad de descarga en
megabits por segundo, y mostrar el tiempo de descarga en minutos y segundos
'''
T=int(input("Ingrese el tamaño del archivo en Megabytes: "))
V=int(input("Ingrese la velocidad de descarga en megabits por segundo: "))
Tm=(T*8)/V
m=int(Tm//60)
s=int(Tm%60)
print("El tiempo de descarga será de ",m," minutos y ",s,"segundos")

File: JAVA/Ejercicio2.java
import java.util.Scanner;
public class Ejercicio2 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Ingrese el tamaño del archivo en Megabytes: ");
        int T = sc.nextInt();
        System.out.print("Ingrese la velocidad de descarga en megabits por segundo: ");
        int V = sc.nextInt();
        int Tm = (T * 8) / V;
        int m = Tm / 60;
        int s = Tm % 60;
        System.out.println("El tiempo de descarga será de " + m + " minutos y " + s + " segundos");
    }
}
